django_simple_bulma/extensions/bulma-dropdown/dist/js/bulma-dropdown.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+    function handle_event_wrapper(show, hide) {
+        const handle_event = (e) => {
+            if (show()) {
+                hide();
+            }
+            e.stopPropagation();
+        }
+
+        return handle_event;
+    }
+
+    let elements = document.getElementsByClassName("dropdown");
+
+    for (let element of elements) {
+
+        // We shouldn't handle hoverable menues
+        if (element.classList.contains("is-hoverable")) {
+            continue;
+        }
+
+        const show = () => {
+            let active = false;
+            if (element.classList.contains("is-active")) {
+                active = true;
+            }
+            element.classList.add("is-active");
+            return active;
+        }
```

Comment:
this looks better, but if I'm reading this correctly, if the element already has the `is-active` class, we still try to add that class before returning `true`. What happens when you try to add a class that an element already has - will it end up with two of that class, or will it just do nothing?

Either way, it would be tidier if it just returned `true` right away, so here's a suggestion for how that might look.

```suggestion
        const show = () => {
            if (element.classList.contains("is-active")) {
                return true;
            }
            element.classList.add("is-active");
            return false;
        }
```